\documentclass{article}
\usepackage[utf8]{inputenc}
\usepackage{amsthm}
\usepackage{amsmath}
\usepackage{verbatim}
\DeclareMathOperator*{\argminA}{arg\,min}
\title{PhD Project - 1st Progress Report}
\author{Jakub Koubele}
\date{\today}
\usepackage{natbib}
\usepackage{graphicx}
\usepackage{amsthm}
\usepackage{amssymb}
\usepackage[colorlinks=true,
linkcolor=black,
citecolor=black,
urlcolor=blue]{hyperref}

%\newtheorem{theorem}{Theorem}
\newtheorem{theorem}{Theorem}[section]
\newtheorem{corollary}{Corollary}[theorem]
\newtheorem{lemma}[theorem]{Lemma}
\graphicspath{ {/} }
\theoremstyle{remark}
\newtheorem*{remark}{Remark}

\theoremstyle{definition}
\newtheorem{definition}{Definition}[section]
\newtheorem{example}{Example}[section]

\bibliographystyle{apalike}

\usepackage[utf8]{inputenc}
\usepackage[english]{babel}

\usepackage{blindtext}


\begin{document}
	
	\maketitle
	
	\section{Introduction}
	My PhD project consists of two main parts: 
	\begin{enumerate}
		\item Development of a computational method analyzing changes in RNA elongation speed from total RNA-seq data.
		\item Calling of mismatched in transcript from single-cell RNA-seq data, and integration of it with single-cell data analysis.
	\end{enumerate}
	From my last TAC meeting (where the initial research proposal was discussed) held in the March 2025, I did spend most of my time working on the first part of my PhD project (the elongation speed model).
	
	I developed a statistical model that jointly estimates not only the changes in RNA elongation speed, but also the changes in intron splicing speed. The pre-print of this work is available on BioRxiv \citep{koubeleEstimatingChangesRNA2025a} (\href{https://doi.org/10.1101/2025.08.24.672013}{link}), and the implementation of the model is provided as an open-source software. 
	
	Development of the model turned out to be more challenging than I initially expected, as multiple aspect of the RNA metabolism influence the total RNA-seq data simultaneously. Besides the mentioned speed of intron splicing (for which the model accounts), changes in the RNA degradation rate (i.e. the transcripts half-life) have an impact on the observed data, which complicates the application of the model (this is caused by a general limitation of steady state RNA-seq data). I am currently trying to finalize the development of the model and then submit the paper for a publication.
	
	For the second part of my PhD project (calling of transcript mismatches from single-cell RNA-seq data), I managed to conduct only a brief analysis of a dataset of mice with knock-down in the Ercc1 gene. As this preliminary analysis didn't shown any significant difference in the number of somatic mutations between the knockdown and control samples, I decided to focus on the development of the elongation speed model for now.
	
	\section{Statistical model of changes in transcription elongation and intron splicing speeds}
	This part of my PhD project is a follow-up on the work conducted previously by other members of the Beyer lab \citep{debesAgeingassociatedChangesTranscriptional2023}. I initially intended to develop a statistical model for measuring changes in transcription elongation speed, that would improve over the previous work in the following aspects:
	\begin{itemize}
		\item I wanted to set the model to a statistical framework similar to how differential expression analysis is commonly conducted, e.g. as in the DESeq2 package \citep{loveModeratedEstimationFold2014}. That is, the model estimates effects of arbitrary covariates (e.g., the treatment effect, but also the effect of sources of unwanted variation such as sequencing batch), and contrasts of interests are statistically tested. This is performed on level of individual genes or introns; downstream analysis can be then performed, e.g., gene set enrichment analysis of genes with statistically significant increase in transcription elongation speed.
		\item Previous work utilized measurement of read coverage in introns to estimate the changes in elongation speed. However, this measurement is influenced not only by changes in the elongation speed, but also by changes in the gene expression. To distinguish the effect caused by changes in elongation speed only, I intended to estimate gene expression changes by accounting also for changes in exon read counts , which is mostly not influenced by changes in elongation speed.
	\end{itemize}	
	I successfully implemented the model with respect to the first goal. The model explicitly specifies the probability distribution of the observed data (conditioned on the model parameters), and the parameters are estimated by the maximum likelihood estimation (MLE). Hypotheses (contrasts of interests) are then tested by the likelihood-ratio test. 
	
	The implementation is done with the usage of PyTorch library \citep{anselPyTorch2Faster2024}, making use of its automatic differentiation system for model fitting. The surrounding workflow, from pre-processing of the input data to the model fitting, is orchestrated with Nextflow \citep{ditommasoNextflowEnablesReproducible2017}.
	
	With respect to the second goal - accounting for the changes in gene expression - I realized that the landscape of relevant aspects of RNA metabolism is even more complex. Currently, I believe that the following four aspects of RNA metabolism need to be considered jointly, in order to correctly estimate the changes in the transcription elongation speed:
	\begin{enumerate}
		\item Gene expression (how often is given gene being transcribed)
		\item Transcription elongations speed
		\item Speed of intron splicing
		\item Degradation rate of mature RNA
	\end{enumerate}  

	In this progress report, I provide a schematic illustrations of how these different aspects of RNA metabolism influence the observed data. For a more technical description, please see the bioRxiv pre-print.
	
	The method works with a total RNA-seq, which is a combination of nascent and mature RNA (see Figure \ref{fig:total_rna}). Changes in different aspects of RNA metabolism then affect the composition of the RNA pool present in a given sample (see Figures \ref{fig:gene_expression} - \ref{fig:rna_degradation}).
	
	\begin{figure}[htbp]
		\centering
		\includegraphics[width=1.0\textwidth]{figures/total_rna.png}
		\caption{Total RNA is a combination of nascent and mature RNA. Intronic RNA (in blue) may come from two distinct sources: either the intron is currently being transcribed by RNA polymerase, or it was already finished, but not spliced and degraded yet. The finished introns may still be part of the nascent RNA or may be retained introns in the mature RNA.}
		\label{fig:total_rna}
	\end{figure}
	
	\begin{figure}[htbp]
		\centering
		\includegraphics[width=1.1\textwidth]{figures/gene_expression.png}
		\caption{Changes in gene expression (how often is a given gene transcribed) increases the abundance of both nascent and mature RNA of that gene.}
		\label{fig:gene_expression}
	\end{figure}
	
	\begin{figure}[htbp]
		\centering
		\includegraphics[width=1.1\textwidth]{figures/elongation_speed.png}
		\caption{Effect of the change in the elongation speed. In the condition with faster RNA polymerase (right), there is less polymerases currently transcribing a gene, compared to the	condition with slower polymerase (left). Note that the amount of mature RNA is
			not affected by the change in elongation speed.}
		\label{fig:elongation_speed}
	\end{figure}
	
	\begin{figure}[htbp]
		\centering
		\includegraphics[width=1.1\textwidth]{figures/splicing_time.png}
		\caption{Splicing speed influences the number of fully transcribed introns that are not spliced and degraded yet; with slower splicing (right), the sample will contain more of such transcribed introns. As we are rellying on the short read RNA sequencing, we are not able to distinguish unspliced introns in the nascent RNA from the retained introns in the mature RNA.}
		\label{fig:splicing_time}
	\end{figure}
	
	\begin{figure}[htbp]
		\centering
		\includegraphics[width=1.1\textwidth]{figures/rna_degradation.png}
		\caption{RNA degradation rate influence the amount of mature RNA, but not of the nascent RNA. With faster degradation (right), less mature RNA will be present in the sample.}
		\label{fig:rna_degradation}
	\end{figure}
	
	Changes in each aspect of the RNA metabolism listed above produces different change in the RNA content of the sample and consequently in the sequencing data. In my model, the log fold changes (LFCs) of different aspects of the RNA metabolism are the parameters being fitted and statistically tested. As there is no observable quantity (e.g., the number of intronic reads) that would be affected by the elongation speed only, the model needs to consider all other aspects of RNA metabolism, in order to disambiguate which of them can be used to explain the observed data.
	
	However, I recently realized that the estimation of all LFCs of interest is a subject to the following limitation: the total RNA-seq data used in the model are usually assumed to be from a steady state condition (i.e., even thought the dataset contains samples with different covariates, such as treatment and control group, we are not observing a time-course data of a samples subjected to some sort of sudden perturbation). However, such steady-state observation is invariant to a scaling of all metabolic rates by a positive constant. If, for example, samples in the treatment group would have $2\times$ higher gene expression rate, but also $2\times$ faster transcription elongation, intron splicing and RNA degradation, no changes would be observable (using total RNA-seq) with comparison to the control group.
	
	Consequently, the LFCs of all aspects of the RNA metabolism cannot be simultaneously estimated using total RNA-seq data; one rate must be left as a baseline, to which the estimation of all remaining rates would be related. In my model, the RNA degradation rate is not included as a model parameter, and only changes in the gene expression, transcription elongation and intron splicing are estimated explicitly. However, as the RNA degradation rate may actually change between samples, such change would influence the remaining LFCs estimated by the model.
	
	Due to this limitation inherent to total RNA-seq data, I am currently discussing (with my supervisor) the option to conduct experiment(s) using RNA labeling method. Such experiment (performed together with the total RNA-seq) would allow us to overcome the limitation of steady-state observations and correctly estimate changes in all RNA metabolism rates of interest (for a particular dataset).
	 
	  

	
	
       
	
	\bibliography{references}
	
		
	\end{document}
